packages/vuetify/src/components/VCombobox/VCombobox.js

```diff
@@ -44,7 +44,7 @@ export default {
     menuCanShow () {
```

Comment:
Since changes are already being made here, we can (and probably should) move this logic to `v-autocomplete` and remove its re-declaration here.

Reply:
Could you, please, explain it more. (Because behaviour of `v-autocomplete` and `v-combobox` supposed to be different when `hideNoData` is `false`, there are no items to display, and there is no `no-data` slot. In `v-autocomplete` menu with standard message appears (`No data available`), and in `v-combobox` no menu appears)